cdn/storedriver/local/local_driver_test.go

```diff
@@ -18,7 +18,6 @@ package local
 
 import (
 	"io"
-	"io/ioutil"
 	"os"
 	"path/filepath"
 	"strings"
```

Comment:
@gaius-qi The CI failure is weird because `io` and `os` are indeed imported correctly. 

Is it because `sudo make test-coverage` in `ci.yml` is not using Go 1.17?

Reply:
CI uses golang 1.17.

https://github.com/Juneezee/Dragonfly2/blob/refactor/deprecate-ioutil/.github/workflows/ci.yml#L12

Reply:
@gaius-qi I tested this on my fork repository. You can see the job log here https://github.com/Juneezee/Dragonfly2/runs/4467635006?check_suite_focus=true

The screenshot below shows that running `go` without `sudo` uses Go 1.17
![image](https://user-images.githubusercontent.com/20135478/145356234-e0079b01-4283-4371-9262-244e2ed13724.png)

---

However, running `go` with `sudo` will use Go 1.15
![image](https://user-images.githubusercontent.com/20135478/145356260-4561d332-5db0-4153-8f40-11e833457526.png)